pkg/mapper/mapper.go

```diff
@@ -70,19 +71,37 @@ var defaultQuantiles = []metricObjective{
 	{Quantile: 0.99, Error: 0.001},
 }
 
+// GetDefaultHistogramOptions returns the default HistogramOptions with default buckets if none are present
```

Comment:
Do these functions need to make a copy of the default structs and return that since the registry uses these functions to get the options and then overwrites them?